noggin/controller/__init__.py

```diff
@@ -0,0 +1,21 @@
+from flask import Blueprint, g, render_template, session
+
+from noggin.utility.templates import gravatar
+
+
+blueprint = Blueprint("root", __name__)
```

Comment:
I think `blueprint` is a little too generic (could be `root` to match the name of the blueprint), but this shouldn't block merging.